pkg/plugins/golang/v2/api.go

```diff
@@ -126,29 +128,29 @@ func (p *createAPISubcommand) Run() error {
 }
 
 func (p *createAPISubcommand) Validate() error {
-	if err := p.resource.ValidateV2(); err != nil {
+	if err := p.options.Validate(); err != nil {
```

Comment:
Note that we have validation for v2 and another for v3. Then, are we still respecting it?